README.md

```diff
@@ -124,14 +124,13 @@ function getDriveService() {
 
       // Sets the login hint, which will prevent the account chooser screen
       // from being shown to users logged in with multiple accounts.
+      // Prefer Session.getEffectiveUser().getEmail() if used in add-ons.
```

Comment:
I think `getEffectiveUser()` is likely a better solution all the time. Instead of adding this in a comment, can you just change the code to use it?